test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
it's well done by listing this file in gitignore , but the right why to do this is by deleting it adding it to gitignore and creating it again because since it's ignored by git it should not appear here in github.

Reply:
Resolved....Thanks for the feedback